src/components/avatar/Avatar.tsx

```diff
@@ -67,13 +86,17 @@ function Avatar({ avatarUrl, className, name, id, isExternal, shouldShowExternal
         avatar = <UnknownUserAvatar className="avatar-icon" />;
     }
 
+    let badge = null;
+    if (shouldShowExternal && isExternal) {
+        badge = <GlobeBadge16 className="bdl-Avatar-badge bdl-Avatar-externalBadge" />;
```

Comment:
It's probably safest to style this separately and leave `bdl-Avatar-badge` off of it.

Reply:
`bdl-Avatar-badge` reuses most generic part of the styling there were in `bdl-Avatar-externalBadge`. I've reused it for other badges but didn't wanted to mess with more specific styling.

I guess I could do it like so:
```css
.bdl-Avatar-badge,
.bdl-Avatar-externalBadge {
    background-color: $white;
    border-color: $white;
    border-style: solid;
    border-radius: 50%;
}
```

and then this line would be
```suggestion
        badge = <GlobeBadge16 className="bdl-Avatar-externalBadge" />;
```